webpack.config.babel.js

```diff
@@ -179,11 +179,11 @@ module.exports = ({ stats = false } = {}) => ({
       // AUTOCONNECT: only applies for mock implementation
       AUTOCONNECT: 'true',
       PRICE_ESTIMATOR_URL: process.env.PRICE_ESTIMATOR_URL || 'develop',
-      APP_ID: process.env.APP_ID,
-      INFURA_ID: process.env.INFURA_ID,
-      WALLET_CONNECT_BRIDGE: process.env.WALLET_CONNECT_BRIDGE,
-      ETH_NODE_URL: process.env.ETH_NODE_URL,
-      LIQUIDITY_TOKEN_LIST: process.env.LIQUIDITY_TOKEN_LIST,
```

Comment:
Question: env values are still read in case they are passed in, right?
Only when not set they receive null, correct?

Reply:
yes, basically `process.env.LIQUIDITY_TOKEN_LIST = process.env.LIQUIDITY_TOKEN_LIST || null`